mdk/commands/phpunit.py

```diff
@@ -103,6 +103,14 @@ class PhpunitCommand(Command):
                 'metavar': 'filter'
             }
         ),
+        (
+            ['--repeat'],
+            {
+                'default': None,
+                'help': 'run tests repeatedly',
```

Comment:
If you intend for the repeat to be limited to a certain number of repeats, please clarify it in the help. Not, however, that it makes it more difficult for the user (and the parser) when a flag can have an optional flag, e.g. `--repeat`, `--repeat 20`.

I would advocate for another flag which sets the number of repeats. 

```
--repeat Repeats the test after running them
--repeat-no {int} The number of times to repeat the test
```

The second one may implicitly assume that the former is true.

Reply:
I updated the help text to be a bit more descriptive. My intention is that it should take a number, similar to PHPUnit's `--repeat` option.